### ISSUE TITLE
Auth stopped returning name, email etc.

### ISSUE THREAD

**Issue by hammadtq** [blockstack/blockstack-ios#53]
It was working fine just a few hours ago using the current master branch (returning name of the user after auth), however, the name and other details started coming back as nil. 

I have just now updated to multiplayer-storage branch and the issue is still there.

Here is the obfuscated dump of retrievedUserData:

> Optional(Blockstack.Payload(jti: Optional("xxxxxxx-xxxx-xxxxx"), iat: Optional(xxxxxxx), exp: Optional(xxxx5940), iss: Optional("did:btc-addr:xxxxxxxxxxx"), privateKey: Optional("xxxxxxxxxxx"), publicKeys: Optional(["xxxxx"]), username: nil, email: nil, coreToken: nil, profileURL: Optional("https://gaia.blockstack.org/hub/xxxxxxx/profile.json"), hubURL: Optional("https://hub.blockstack.org"), version: Optional("1.2.0"), claim: nil, profile: Optional(Blockstack.Profile(type: Optional("Person"), context: Optional("http://schema.org"), name: nil, description: nil, apps: Optional(["https://xxxxx.co/:8080": xxxxxxxxxxxxxx"])))))

It rightly goes into the "Nameless User" check, however, is there anything I can do it make it go away?

**Comment by shreyasthiagaraj** [comment]
@larrysalibra This doesn't sound like an iOS issue. Where does this belong?

**Comment by larrysalibra** [comment]
I'm not sure either. 

@hammadtq is this on a newly created Blockstack ID?

**Comment by hammadtq** [comment]
@larrysalibra made it in April 2018 and registered my name on June 14th. It may not be related to iOS as after this posting this issue I discovered I am getting partial data in the blockstack browser as well. Here is the thread on the forum with more details: https://forum.blockstack.org/t/auth-returning-incomplete-data/6082

**Comment by hammadtq** [comment]
@shreyasthiagaraj dug further, basically, iOS auth is refreshing the "claim" tag in profile.json. It leaves apps, type, context but removes the name, description etc. I have tried using auth using my old [fork](https://github.com/hammadtq/blockstack-ios) that worked fine.

I have also tried looking at the new auth code, payload dictionary seems to be fine. I can see "claims:" in a construct in Tokens.swift and blockstack.js but seems like auth.makeRequest is not using them. I am not aware of the whole architecture yet. What can be the problem?

I have updated my debugging on the forum [thread](https://forum.blockstack.org/t/auth-returning-incomplete-data/6082/6).
